dotcom-rendering/src/model/enhanceCollections.ts

```diff
@@ -101,7 +101,34 @@ const enhanceCards = (
 const decideContainerPalette = (
 	metadata?: { type: FEContainerPalette }[],
 ): DCRContainerPalette | undefined => {
-	return metadata?.length && metadata[0].type ? metadata[0].type : undefined;
+	if (!metadata?.length || metadata[0].type) return undefined;
+	switch (metadata[0].type) {
```

Comment:
```suggestion
	switch (metadata?.[0]?.type) {
```

Reply:
<img width="457" alt="image" src="https://user-images.githubusercontent.com/9575458/168109657-8da0c484-bb26-4196-8420-ca0f071b3342.png">
Unfortunately you're too smart for typescript!

Reply:
Done :)